src/ast/nodes/AssignmentExpression.ts

```diff
@@ -46,6 +41,13 @@ export default class AssignmentExpression extends NodeBase {
 		return path.length > 0 && this.right.hasEffectsWhenAccessedAtPath(path, context);
 	}
 
+	include(context: InclusionContext, includeChildrenRecursively: IncludeChildren): void {
+		if (!this.deoptimized) this.applyDeoptimizations();
+		this.included = true;
+		this.left.include(context, includeChildrenRecursively);
+		this.right.include(context, includeChildrenRecursively);
+	}
```

Comment:
Similar reasons, we could also call `super.include`, but this is all it would do, though `Node.include` is slightly more complicated and generic, so this is more efficient.